resources/views/platform/components/game/current-progress/big-status-label.blade.php

```diff
@@ -35,15 +37,35 @@
     $statusLabel = "Awarded";
 }
 
+$isBeaten = $isBeatable && ($isBeatenHardcore || $isBeatenSoftcore);
+
 // This case can occur on legacy completion/mastery awards
 // where progression achievements were added after the user
 // had already mastered the game. It's an edge case, but we
 // try to gracefully handle it anyway.
-if (($isCompleted || $isMastered) && ($isBeatable && !$isBeatenHardcore && !$isBeatenSoftcore)) {
+if (($isCompleted || $isMastered) && !$isBeaten) {
```

Comment:
Not sure if it's this line, but something in this PR marks a game as Unbeaten if it's been mastered but there aren't any progression achievements.
![image](https://github.com/RetroAchievements/RAWeb/assets/32680403/8923c28c-83c5-42df-a977-bd5cb39a2177)

Reply:
Good catch, fixed in https://github.com/RetroAchievements/RAWeb/pull/1859/commits/bfbca79c9b38b3e3947c8f6a41d20e43b580fb58.